Dockerfiles/manifests/aws/tf/README.md

```diff
@@ -35,3 +35,16 @@ To allow kubectl to talk to your cluster `make kubeconfig` and follow the sugges
 Make sure nodes can register `make config-map-aws-auth` and wait for nodes to appear.
 
 Your are now ready.
+
+### Setup on an already provisioned Cluster
+
+Get the kubectl command and aws-iam-authenticator command
+Get the kubeconfig file form someone and put it somewhere
```

Comment:
`Get the kubeconfig file form someone and put it somewhere` 😆

Reply:
Maybe we should put the kubeconfig file in keybase? And you should put in the `~/.kube` folder

Reply:
the default filename is `config` but it's also possible to switch config files on the fly

Reply:
In reality config file might be constructed using  https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html  ...  "To create your kubeconfig file"

Reply:
If you have terraform tf state file for cluster (which can be either stored somewhere or committed)

We have two actions in terraform scripts:  `make kubelet-config` - to get the config.

...

and `config-map-aws-auth` to initialize cluster once.

```
kubelet-config:
        terraform output kubeconfig > kubeconfig
        export KUBECONFIG=$(PWD)/kubeconfig

config-map-aws-auth:
        terraform output config-map-aws-auth > config-map-aws-auth.yaml
        kubectl apply -f config-map-aws-auth.yaml
        kubectl get nodes --watch

```

Reply:
I like all your suggestions, I've just written down how I got setup. I am assuming others are setup in the same way. Goal of my work now is not to improve this process, just document how it was done. If we want to improve the setup (e.g. version the terraform file on s3 for all to use and derive credentials), we can do that but I'm not sure I'm the one who should take lead on this